docs/content/latest/api/ysql/the-sql-language/statements/cmd_analyze.md

```diff
@@ -0,0 +1,139 @@
+To collect or update statistics the ANALYZE command should be explicitly executed.
+{{< /note >}}
+
+## Syntax
+
+<ul class="nav nav-tabs nav-tabs-yb">
+  <li >
+    <a href="#grammar" class="nav-link active" id="grammar-tab" data-toggle="tab" role="tab" aria-controls="grammar" aria-selected="true">
+      <i class="fas fa-file-alt" aria-hidden="true"></i>
+      Grammar
+    </a>
+  </li>
+  <li>
+    <a href="#diagram" class="nav-link" id="diagram-tab" data-toggle="tab" role="tab" aria-controls="diagram" aria-selected="false">
+      <i class="fas fa-project-diagram" aria-hidden="true"></i>
+      Diagram
+    </a>
+  </li>
+</ul>
+
+<div class="tab-content">
+  <div id="grammar" class="tab-pane fade show active" role="tabpanel" aria-labelledby="grammar-tab">
+    {{% includeMarkdown "../../syntax_resources/the-sql-language/statements/analyze,table_and_columns.grammar.md" /%}}
+  </div>
+  <div id="diagram" class="tab-pane fade" role="tabpanel" aria-labelledby="diagram-tab">
+    {{% includeMarkdown "../../syntax_resources/the-sql-language/statements/analyze,table_and_columns.diagram.md" /%}}
+  </div>
+</div>
+
+## Semantics
+
+### VERBOSE
+
+Enable display of progress messages.
+
+### *table_name*
+
+Specify the table, optionally schema-qualified, to be analyzed. If not specified, then all regular tables in the current database will be analyzed.
+
+### *column_name*
+
+Specify the list of columns to be analyzed. If not specified, then all columns of the table will be analyzed.
+
+## Examples
+
+### Analyze single table
+
+```plpgsql
+yugabyte=# ANALYZE some_table;
+```
+
+```output
+ANALYZE
+```
+
+### Analyze some columns of the table
+
+```plpgsql
+yugabyte=# ANALYZE some_table(col1, col3);
+```
+
+```output
+ANALYZE
+```
+
+### Analyze multiple tables verbosely
+
+```plpgsql
+yugabyte=# ANALYZE VERBOSE some_table, other_table;
+```
+
+```output
+INFO:  analyzing "public.some_table"
+INFO:  "some_table": scanned, 3 rows in sample, 3 estimated total rows
+INFO:  analyzing "public.other_table"
+INFO:  "other_table": scanned, 3 rows in sample, 3 estimated total rows
+ANALYZE
+```
+
+### Analyze affects query plans
+
+In the absence of the statistics optimizer uses hard-coded defaults, such us 1000 for the number of rows in the table.
+After ANALYZE number of rows is accurate.
+
+```output
+yugabyte=# CREATE TABLE test(a int primary key, b int);
+CREATE TABLE
+yugabyte=# INSERT INTO test VALUES (1, 1), (2, 2), (3, 3);
+INSERT 0 3
+yugabyte=# EXPLAIN select * from test where b = 1;
+                       QUERY PLAN
+---------------------------------------------------------
+ Seq Scan on test  (cost=0.00..102.50 rows=1000 width=8)
+   Filter: (b = 1)
+(2 rows)
+
+yugabyte=# ANALYZE test;
+ANALYZE
+yugabyte=# EXPLAIN select * from test where b = 1;
+                     QUERY PLAN
+----------------------------------------------------
+ Seq Scan on test  (cost=0.00..0.31 rows=3 width=8)
+   Filter: (b = 1)
+(2 rows)
+```
```

Comment:
Separating inputs from outputs for copy-paste ease...

````suggestion
```sql
yugabyte=# CREATE TABLE test(a int primary key, b int);
```

```output
CREATE TABLE
```

```sql
yugabyte=# INSERT INTO test VALUES (1, 1), (2, 2), (3, 3);
```

```output
INSERT 0 3
```

```sql
yugabyte=# EXPLAIN select * from test where b = 1;
```

```output
                       QUERY PLAN
---------------------------------------------------------
 Seq Scan on test  (cost=0.00..102.50 rows=1000 width=8)
   Filter: (b = 1)
(2 rows)
```

```sql
yugabyte=# ANALYZE test;
```

```output
ANALYZE
```

```sql
yugabyte=# EXPLAIN select * from test where b = 1;
```

```output
                     QUERY PLAN
----------------------------------------------------
 Seq Scan on test  (cost=0.00..0.31 rows=3 width=8)
   Filter: (b = 1)
(2 rows)
```
````

Reply:
In that paragraph I wanted to emphasize sequence and continuity of the statements.  
It is OK to rewrite them as statement/output pairs, however in this case I'd prefer to comment each statement, rather than have single text block explanation.
Will rewrite.